build/configs/rtl8721csm/hello/Make.defs

```diff
@@ -0,0 +1,203 @@
+###########################################################################
+#
+# Copyright 2018 Samsung Electronics All Rights Reserved.
```

Comment:
Could you check the license? Do you want to add Samsung license, not yours? And even if you do, could you change the  year? It should be 2020.

Reply:
We are working on this now, will update the PR soon

Reply:
@prashantrar I saw two more commits. Is it done? But the first file has 2018 as license year when I open the file changed.

Reply:
The license part is pending, most likely I will push one more commit today with the fixed licenses, the change related to make dist clean is done.

Reply:
@sunghan-chang the latest commit has fixed the license info changes.